.github/workflows/scribble_build-guide.yml

```diff
@@ -0,0 +1,29 @@
+name: Scribble Racket Build Guide
+
+on:
+  push:
+    paths:
+      - ".github/workflow/scribble_build-guide.yaml"
+      - "pkgs/racket-build-guide/**"
+  pull_request:
+    paths:
+      - ".github/workflow/scribble_build-guide.yaml"
+      - "pkgs/racket-build-guide/**"
+
+jobs:
+  scribble:
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
Does it really needs to fetch all the commits?

Reply:
ahah - well caught. For some reason, I thought that it was just the last one. But for that I need 1, which is the default so I can remove the line. Thanks.